docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,225 @@
+  :ref:`SASL for authentication <kafka_sasl_auth>`, and :ref:`Authorization with ACLs <kafka_authorization>`.
+- KSQL supports :ref:`Schema Registry security features <schemaregistry_security>` such as SSL and SASL.
+
+To configure security for KSQL, add your configuration settings to the ``<path-to-confluent>/etc/ksql/ksql-server.properties``
+file and then :ref:`start the KSQL server <install_ksql-server>` with your configuration file specified.
+
+.. code:: bash
+
+    $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+.. contents:: Table of Contents
+    :local:
+
+Configuring KSQL for |ccloud|
+-----------------------------
+
+You can use KSQL with a Kafka cluster in |ccloud|. For more information, see :ref:`install_ksql-ccloud`.
+
+.. _config-security-ksql-sr:
+
+Configuring KSQL for Secured Confluent Schema Registry
+------------------------------------------------------
+
+The following configuration is connecting KSQL with the Confluent Schema Registry over HTTPS.
+
+#. Specify the HTTPS endpoint in the ``ksql.schema.registry.url`` setting in the
+   KSQL server configuration file:
+
+   .. code:: bash
+
+        ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+
+#. If the Schema Registry's SSL certificate is not signed by a CA that is recognized by the JVM by default, then you
+   must provide a suitable truststore when starting KSQL via the command line:
+
+   .. code:: bash
+
+      $ KSQL_OPTS="-Djavax.net.ssl.trustStore=<path-to-trust-store> -Djavax.net.ssl.trustStorePassword=<store-password>" ksql-server-start <props>
+
+The exact settings you need depend on what SASL mechanism the Confluent Schema Registry is using is using and how your SSL certificates
+are signed. For more information, see :ref:`schemaregistry_security`.
+
+.. _config-security-kafka:
+
+Configuration KSQL for Secured Apache Kafka clusters
+----------------------------------------------------
+
+To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. The following
+are common configuration examples.
+
+.. _config-security-ssl-sasl:
+
+----------------------------------------------------------
+Configuring Encrypted Communication between KSQL and Kafka
+----------------------------------------------------------
+
+You can connect to a secure Kafka cluster using PLAIN SASL and SSL, where the certificates have been signed by a CA trusted by
+the default JVM trust store.
+
+This configuration enables KSQL to connect to a secure Kafka cluster using PLAIN SASL, where the SSL certificates have been
+signed by a CA trusted by the default JVM trust store. Other options include GSSAPI and Kerberos.
+
+.. code:: bash
+
+    security.protocol=SASL_SSL
+    sasl.mechanism=PLAIN
+    sasl.jaas.config=\
+        org.apache.kafka.common.security.plain.PlainLoginModule required `
+        username="<authenticated-ksql-user>" `
```

Comment:
drop the `authenticated`. This is the config that will hopefully allow the user to _become_ authenticated.